packages/cli/src/cli.ts

```diff
@@ -0,0 +1,146 @@
+import { ArgumentParser } from "argparse";
+// tslint:disable-next-line:no-submodule-imports
+import colors = require("colors/safe");
+import { join } from "path";
+
+import { TsRepl } from "./tsrepl";
+
+export function main(originalArgs: readonly string[]): void {
+  const parser = new ArgumentParser({ description: "The CosmWasm REPL" });
+  parser.addArgument("--version", {
+    action: "storeTrue",
+    help: "Print version and exit",
+  });
+
+  const maintainerGroup = parser.addArgumentGroup({
+    title: "Maintainer options",
+    description: "Don't use those unless a maintainer tells you to.",
+  });
+  maintainerGroup.addArgument("--selftest", {
+    action: "storeTrue",
+    help: "Run a selftext and exit",
+  });
+  maintainerGroup.addArgument("--debug", {
+    action: "storeTrue",
+    help: "Enable debugging",
+  });
+  const args = parser.parseArgs([...originalArgs]);
+
+  if (args.version) {
+    const version = require(join(__dirname, "..", "package.json")).version;
+    console.info(version);
+    return;
+  }
+
+  const imports = new Map<string, readonly string[]>([
+    ["@cosmwasm/sdk", ["types", "RestClient"]],
+    [
+      "@iov/crypto",
```

Comment:
These three packages are not part of the dependencies. (crypto, encoding, keycontrol). This is related to my questions above. How do we pull all this in?